Content.Client/PlantAnalyzer/UI/PlantAnalyzerBoundUserInterface.cs

```diff
@@ -0,0 +1,53 @@
+using Content.Shared.PlantAnalyzer;
+using JetBrains.Annotations;
+
+namespace Content.Client.PlantAnalyzer.UI;
+
+[UsedImplicitly]
+public sealed class PlantAnalyzerBoundUserInterface : BoundUserInterface
+{
+    [ViewVariables]
+    private PlantAnalyzerWindow? _window;
+
+    public PlantAnalyzerBoundUserInterface(EntityUid owner, Enum uiKey) : base(owner, uiKey)
+    {
+    }
+
+    protected override void Open()
+    {
+        base.Open();
+        _window = new PlantAnalyzerWindow(this)
+        {
+            Title = Loc.GetString("plant-analyzer-interface-title"),
```

Comment:
I don't think this is needed since title is in the .xaml?

Reply:
My memory tells me at some point it didn't display the title if you opened the UI manually instead of after a scan, and this was needed. Testing now, it does seem unnecessary.